samples/explore_datasource.py

```diff
@@ -9,56 +9,61 @@
 # on top of the general operations.
 ####
 
-
-import tableauserverclient as TSC
-import os.path
 import argparse
 import getpass
 import logging
 
+import tableauserverclient as TSC
+
+
 parser = argparse.ArgumentParser(description='Explore datasource functions supported by the Server API.')
 parser.add_argument('--server', '-s', required=True, help='server address')
 parser.add_argument('--username', '-u', required=True, help='username to sign into server')
 parser.add_argument('--publish', '-p', metavar='FILEPATH', help='path to datasource to publish')
 parser.add_argument('--download', '-d', metavar='FILEPATH', help='path to save downloaded datasource')
 parser.add_argument('--logging-level', '-l', choices=['debug', 'info', 'error'], default='error',
                     help='desired logging level (set to error by default)')
-args = parser.parse_args()
 
-password = getpass.getpass("Password: ")
 
-# Set logging level based on user input, or error by default
-logging_level = getattr(logging, args.logging_level.upper())
-logging.basicConfig(level=logging_level)
+if __name__ == '__main__':
+
+    args = parser.parse_args()
+
+    password = getpass.getpass("Password: ")
+
+    # Set logging level based on user input, or error by default
+    logging_level = getattr(logging, args.logging_level.upper())
+    logging.basicConfig(level=logging_level)
 
-# SIGN IN
-tableau_auth = TSC.TableauAuth(args.username, password)
-server = TSC.Server(args.server)
-with server.auth.sign_in(tableau_auth):
-    # Query projects for use when demonstrating publishing and updating
-    all_projects, pagination_item = server.projects.get()
-    default_project = next((project for project in all_projects if project.is_default()), None)
+    # SIGN IN
```

Comment:
I'm not sure if this is just a product of github, but the spaces here are showing different than then ones below.  Make sure they are all spaces, not a mix of tabs and spaces.